docs/form.md

```diff
@@ -0,0 +1,12 @@
+[< Back to Components List](../README.md#components)
+
+## Form
+`polaris-form` implements the [Polaris Form component](https://polaris.shopify.com/components/forms/form).
+
+### Examples
+
+```hbs
+{{#polaris-form onSubmit=(action "handleFormSubmit")}}
+  ... items...
+{{/polaris-form}}
+```
```

Comment:
Maybe worth adding one or two more examples here to show usage of different properties, as we've done with other components?

Reply:
didn't found the needed really, since everything is just passing a property and nothing special